skipruntime-ts/ts/src/skipruntime_api.ts

```diff
@@ -297,7 +235,27 @@ export interface EagerCollection<K extends TJSON, V extends TJSON>
    * @returns the value for this `key`, or null if no such value exists
    */
   maybeGetOne(key: K): Opt<V>;
+}
 
+export interface CollectionAccess<K extends TJSON, V extends TJSON>
+  extends CollectionReader<K, V> {
+  /**
+   * Get all values of an eager reactive collection, if one exists.
+   * Exist only in .
```

Comment:
Not sure what this comment is saying.

More generally, could you explain a bit about the relationship between the different kinds of collection interfaces here?  `Writer` vs. `Reader` is intuitive, but I think that the distinction between `Access` and `Reader` is a bit fuzzy